crates/ruff_linter/src/rules/pylint/rules/super_without_brackets.rs

```diff
@@ -0,0 +1,88 @@
+use ruff_python_ast::{self as ast, Expr};
+
+use ruff_diagnostics::{AlwaysFixableViolation, Diagnostic, Edit, Fix};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_text_size::Ranged;
+
+use crate::checkers::ast::Checker;
+
+/// ## What it does
+/// Checks for `super` calls without parentheses.
+///
+/// ## Why is this bad?
+/// When `super` is used without parentheses, it is not an actual call.
+///
+/// ## Example
+/// ```python
+/// class Soup:
+///     @staticmethod
+///     def temp():
+///         print("Soup is hot!")
+///
+///
+/// class TomatoSoup(Soup):
+///     @staticmethod
+///     def temp():
+///         super.temp()  # [super-without-brackets]
+///         print("But tomato soup is even hotter!")
+/// ```
+///
+/// Use instead:
+/// ```python
+/// class Soup:
+///     @staticmethod
+///     def temp():
+///         print("Soup is hot!")
+///
+///
+/// class TomatoSoup(Soup):
+///     @staticmethod
+///     def temp():
+///         super().temp()
+///         print("But tomato soup is even hotter!")
+/// ```
+#[violation]
+pub struct SuperWithoutBrackets;
+
+impl AlwaysFixableViolation for SuperWithoutBrackets {
+    #[derive_message_formats]
+    fn message(&self) -> String {
+        format!("`super` call without parentheses")
+    }
+
+    fn fix_title(&self) -> String {
+        "Add parentheses to `super` call".to_string()
+    }
+}
+
+/// PLW0245
+pub(crate) fn super_without_brackets(checker: &mut Checker, func: &Expr) {
+    if !checker.semantic().current_scope().kind.is_function() {
+        return;
+    }
```

Comment:
We could probably check if we're in a function immediately within a class?

Reply:
Actually, can you try using `function_type::classify` for this, to detect that we're in a method?

Reply:
You will need to pass the parent scope via something like:

```rust
let Some(parent) = &checker.semantic().first_non_type_parent_scope(scope) else {
    return;
};
```

Instead of the current scope.

Reply:
Updated! Seems much better now, LMK!